docs/developer/standards.md

```diff
@@ -0,0 +1,48 @@
+# Standards
```

Comment:
Should be in "contributors" folder per #941

Reply:
Is now.